onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1099,69 +1098,80 @@ with the exception that numpy default keepdims to False instead of True.)DOC")
       .SetDomain(kMSDomain)
       .SinceVersion(1)
       .SetDoc(R"DOC(
-Pruning away boxes that have high intersection-over-union (IOU) overlap with previously selected boxes.
-Bounding boxes with score less than score_threshold are removed. Bounding boxes are supplied as [y1, x1, y2, x2],
-where (y1, x1) and (y2, x2) are the coordinates of any diagonal pair of box corners and the coordinates can be provided
-as normalized (i.e., lying in the interval [0, 1]) or absolute.
+Filter out boxes that have high intersection-over-union (IOU) overlap with previously selected boxes.
+Bounding boxes with score less than score_threshold are removed. Bounding box format is indicated by attribute center_point_box.
 Note that this algorithm is agnostic to where the origin is in the coordinate system and more generally is invariant to
-orthogonal transformations and translations of the coordinate system;
-thus translating or reflections of the coordinate system result in the same boxes being selected by the algorithm.
-The output of this operation is a set of integers indexing into the input collection of bounding boxes representing the selected boxes.
-The bounding box coordinates corresponding to the selected indices can then be obtained using the gather operation.)DOC")
-      .Input(0, "boxes", "An input tensor. 2D tensor with shape [num_boxes, 4]", "T1")
-      .Input(1, "scores", "An input tensor. 1D tensor with shape [num_boxes]", "T1")
-      .Output(0, "selected_indices", "selected indices from the boxes tensor.", "T2")
-      .Output(
+orthogonal transformations and translations of the coordinate system; thus translating or reflections of the coordinate system
+result in the same boxes being selected by the algorithm.
+The selected_indices output is a set of integers indexing into the input collection of bounding boxes representing the selected boxes.
+The bounding box coordinates corresponding to the selected indices can then be obtained using the gather operation. This is mostly used in TF models.
+The boxes output is the filtered boxes which set the filtered boxes to [0, 0, 0, 0]. This is mostly used in Pytoch models.)DOC")
+      .Input(
+          0,
+          "boxes",
+          "An input tensor with shape [num_batches, num_classes, spatial_dimension, 4]. The box data format is indicated by center_point_box.",
```

Comment:
In most Mask/Faster RCNN, yolo algorithm, the boxes are independent from the class, why is there a num_classes in the box input?

Reply:
@wenbingl we regress bounding boxes separately for each class in our Yolo model. I will add the model [here](https://github.com/Microsoft/pytorch_od) soon.

Reply:
@dashesy , cool, you mean the boxes may be different among each class? Do you have any paper or docs to help me refresh my knowledge?

Reply:
@wenbingl I looked again! you are correct, that work is not deployed/published yet. @HectorSVC feel free to simplify this, although you already implemented the channel broadcast.

Reply:
I'll update the schema to remove the class dimension from boxes